packages/components/pages/f-account-info/src/components/_tests/AccountInfo.test.js

```diff
@@ -192,5 +256,29 @@ describe('AccountInfo', () => {
                 });
             });
         });
+
+        describe('onFormSubmit ::', () => {
+            it.skip('should set shouldShowErrorPage flag to true if an error occurs', async () => {
```

Comment:
Are these WIP or coming in a later PR?

Reply:
Need to come in a later PR which adds the POST action to save details.